src/mongoose_hooks.erl

```diff
@@ -131,6 +138,26 @@
          pubsub_delete_node/5,
          pubsub_publish_item/7]).
 
+-spec adhoc_local_items(LServer, Acc, From, To, Lang) -> Result when
```

Comment:
No docs for these functions, maybe add them later

Reply:
True, but I am not sure how adhoc works and what exactly is happening here. I didn't want to spend time investigating to deeply, especially seeing that this hook is not covered by tests.